### PR TITLE
Separate chromatic workflows 

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

Moves each Chromatic run to its own workflow instance and adds dependabot checks to all.

## Why?

As we move towards moving DCR into CSNX, it's helpful to have these things separate. We're also now able to run the workflows in parallel, whether or not thats faster I'm not sure at this stage.

### CODE REVIEW

File: .github/workflows/cr-chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Chromatic 👓
```
Comment (joecowton1) [inline .github/workflows/cr-chromatic.yml:1]:
```suggestion
name: CR Chromatic 👓
```

File: .github/workflows/ar-chromatic.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          # and the target branch. This can cause issues with Chromatic https://www.chromatic.com/docs/turbosnap#github-pullrequest-triggers
+          # Hopefully by checking out the HEAD commit of a PR instead of the merge commit we can avoid some of those issues.
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Checkout - On Push Event
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push'}}
+        with:
+          fetch-depth: 0
+
+      - uses: guardian/actions-setup-node@main
```
Comment (georgeblahblah) [inline .github/workflows/ar-chromatic.yml:28]: I think this action is deprecated now (https://github.com/guardian/actions-setup-node/issues/9)! We can use `actions/setup-node@v3`:

```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
          cache: 'yarn'
```

File: .github/workflows/cr-chromatic.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          # and the target branch. This can cause issues with Chromatic https://www.chromatic.com/docs/turbosnap#github-pullrequest-triggers
+          # Hopefully by checking out the HEAD commit of a PR instead of the merge commit we can avoid some of those issues.
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Checkout - On Push Event
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push'}}
+        with:
+          fetch-depth: 0
+
+      - uses: guardian/actions-setup-node@main
```
Comment (georgeblahblah) [inline .github/workflows/cr-chromatic.yml:28]: similar about using `actions/setup-node` here

**Review by georgeblahblah — COMMENTED** [review]
Thanks Joe! I had a suggestion about using `actions/setup-node@v3` over `guardian/actions-setup-node` which is being deprecated. Interested to hear your thoughts!

File: .github/workflows/ar-chromatic.yml (hunk 2 above)
Comment (joecowton1) [reply .github/workflows/ar-chromatic.yml:28]: Ah interesting - seems like a sensible time to make the change.

File: .github/workflows/cr-chromatic.yml (hunk 3 above)
Comment (joecowton1) [inline .github/workflows/cr-chromatic.yml:28]:
```suggestion
      - uses: actions/setup-node@v3

```

File: .github/workflows/ar-chromatic.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Checkout - On Push Event
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push'}}
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+      # Cache npm dependencies using https://github.com/bahmutov/npm-install
+      # Root yarn installs all workspaces (root, common, dotcom)
+      - uses: bahmutov/npm-install@v1
```
Comment (sndrs) [inline .github/workflows/ar-chromatic.yml:31]: we need to point the nvmrc file with the official node setup, and can do away with the bumatov caching action now too:

```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
          cache: 'yarn'
```

File: .github/workflows/cr-chromatic.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push'}}
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+
+      # Cache npm dependencies using https://github.com/bahmutov/npm-install
+      # Root yarn installs all workspaces (root, common, dotcom)
+      - uses: bahmutov/npm-install@v1
```
Comment (sndrs) [inline .github/workflows/cr-chromatic.yml:32]:
```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
          cache: 'yarn'
```

File: .github/workflows/dcr-chromatic.yml [hunk 6]
```diff
@@ -25,30 +25,12 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: guardian/actions-setup-node@main
+      - uses: actions/setup-node@v3
 
       # Cache npm dependencies using https://github.com/bahmutov/npm-install
       # Root yarn installs all workspaces (root, common, dotcom)
       - uses: bahmutov/npm-install@v1
```
Comment (sndrs) [inline .github/workflows/dcr-chromatic.yml:32]:
```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
          cache: 'yarn'
```

**Pull request merged**